### PR TITLE
Capstone website

### PR DESCRIPTION
**Pull request by SabaAhmad404**
This is my capstone project on Summit global Conference. In this project I created the both versions for desktop and mobile as well for that purpose i use
1) HTML and CSS for the pages
2) use JS for the functionality of buttons (kindly refresh the mobile version of home page to see the (more button).
3) added the linters 
here is the link of my loom video https://www.loom.com/share/9b99b54797a4493eb7e36fb722374010
kindly review my project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 
 ## Live Demo (if available)
-Please use the following [link]() to access the website:  
+Please use the following [link]( https://sabaahmad786.github.io/Capstone-1/) to access the website:
```
Comment (Kamba56) [inline README.md]: - [ ] You are required to record a video presenting the features of the project and add the link to the video to your PR description and readme file. Kindly do that, please.
  - The maximum length of the video is 5 minutes.
  - The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
  - The video contains a demo of the project, showing the pages in different screen sizes.
  - The video contains a highlight about a piece of the code that you find interesting or are proud of making.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+                <p>Create opportunities to freely network with CC personnel from around the world.</p>
+            </div>
+        </div>
+        <button class="nature">Summit 2023</button>
+        <a class="seemore">View  Full  Program</a>
+       </section>
+
+
+       <section class="speakrs">
+       </section>
```
Comment (Kamba56) [inline index.html:109]: - [x] I think on the mobile version of the speakers' section, not all the speakers should be visible and then there should be a button that can show more speakers when clicked. You don't have to implement the button functionalities though.
### Your design


### Design guidelines
f2

File: index.html [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+                <h1>Creative Commons</h1>
+                <h1>Global Summit</h1>
+                 <h1>2022</h1>
+            </div>
+            <p class="event-place">A fun festival where people from more than 80 countries who believe in the value of sharing and openness and create positive change gather and share ideas will be held in Korea in October.</p>
+            <p class="Date">2022.10.14 (Wed) ~ 17 (Sat)</p>
+            <p class="place-name">@ National Museum of Korea, Art Center Nabi, etc.</p>
+        </div>
+       </section>
+
```
Comment (Kamba56) [inline index.html]: - [x] Please use an indentation of two spaces in all your documents. This is a convention and also [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

File: index.html [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+       <section id="home-page">
+        <div class="hamburg">
+            <img id="btn-union" src="./images/hamburger-icon.svg" alt="ham">
+        </div>
+        <div class="heading">
+            <div class="Title">
+                <h2 class="Hello">"Hello, Sharing World!"</h2>
+                <h1>Creative Commons</h1>
+                <h1>Global Summit</h1>
+                 <h1>2022</h1>
```
Comment (Kamba56) [inline index.html]: - [x] Please note that you should have only one `h1` in the whole of your document, kindly change this to any other heading tag(from `h2` to `h5`). This is SEO best practice. Kindly check [here](https://yoast.com/one-h1-heading-per-post/) for more information.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,732 @@
...
+  width: 50%;
+}
+
+.grid-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 17px;
+
+  /* margin: 9% 10% 8% 10%; */
```
Comment (Kamba56) [inline style.css]: - [x] Please remove all commented code from your all documents. This will make your document neater.

File: about.html [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+    </header>
+    <main id="header-body">
+        <section id="home-page">
+            <div class="hamburg">
+                <img id="btn-union" src="./images/hamburger-icon.svg" alt="ham">
+            </div>
+            <div class="heading">
+                <div class="Title">
+                    <h1>Creative Commons</h1>
+                  <h1>Global Summit 2022</h1>
```
Comment (Kamba56) [inline about.html]: - [x] Please note that you should have only one `h1` in the whole of your document, kindly change this to any other heading tag(from `h2` to `h5`). This is SEO best practice. Kindly check [here](https://yoast.com/one-h1-heading-per-post/) for more information.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @SabaAhmad786, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html (hunk 3 above)
Comment (SabaAhmad404) [reply index.html]: @Kamba56  i add the button and functionality as well kindly refresh the page and see again..refresh the mobile version

Comment (Kamba56) [reply index.html]: Hi @SabaAhmad786, Kindly request for a re-review if you have implemented all the changes. Don't forget to record the video and add the link.

**Review by omarsalem7 — APPROVED** [review]
Hi @SabaAhmad786 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**